source/dofs/dof_handler_policy.cc

```diff
@@ -3004,10 +3004,10 @@ namespace internal
 
 
 
-        template <int dim, int spacedim>
+        template <int dim, int spacedim, typename DoFHandlerType>
```

Comment:
I have to admit that I'm not a huge fan of template template parameters. They're just so clunky.

Is your concern that the information in the first two, and the third template argument is redundant? I would agree, but I'm not overly worried about this given that it's a completely internal function -- there is no risk of anyone calling it with inconsistent arguments.